pyproject.toml

```diff
@@ -73,6 +73,36 @@ sopel-plugins = "sopel.cli.plugins:main"
 [project.entry-points.pytest11]
 pytest-sopel = "sopel.tests.pytest_plugin"
 
+[tool.flake8]
+max-line-length = 79
+no-accept-encodings = true
+type-checking-exempt-modules = "typing"
+exclude = [
```

Comment:
Do these work differently in pyproject? Just wondering what happened to the trailing `/*`s.

Reply:
Might work even better to use [filename](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-filename) with a couple glob patterns matching the subdirectories we _do_ want (with `.py` suffix included, of course); then the exclusions become trivial.